pkg/host/internal/kernel/kernel.go

```diff
@@ -522,6 +523,29 @@ func (k *kernel) InstallRDMA(packageManager string) error {
 	return nil
 }
 
+func (k *kernel) DiscoverRDMASubsystem() (string, error) {
+	log.Log.Info("DiscoverRDMASubsystem(): retrieving RDMA subsystem mode")
+	subsystem, err := netlink.RdmaSystemGetNetnsMode()
```

Comment:
please use the netlink interface in the project so we can have a mock for it on unit tests